src/Components/OATGraphViewer/Internal/OATGraphCustomEdge.tsx

```diff
@@ -604,31 +613,12 @@ const OATGraphCustomEdge: React.FC<IOATGraphCustomEdgeProps> = ({
                                         fontSize: FontSizes.size10,
                                         color: theme.semanticColors.actionLink,
                                         marginTop: '-35px',
-                                        marginRight: '-10px'
+                                        marginleft: '10px'
                                     }
                                 }}
                             />
                         </ActionButton>
-                        {data.type !== OATExtendHandleName && (
-                            <ActionButton
-                                className={graphViewerStyles.edgeCancel}
-                                onClick={onNameBlur}
-                            >
-                                <Icon
-                                    iconName="Save"
-                                    styles={{
-                                        root: {
-                                            fontSize: FontSizes.size10,
-                                            color:
-                                                theme.semanticColors.actionLink,
-                                            marginTop: '-35px',
-                                            marginRight: '-10px'
-                                        }
-                                    }}
-                                />
-                            </ActionButton>
-                        )}
-                    </body>
```

Comment:
Why has this been removed?

Reply:
The Idea of having the Save button Icon was because when we added the cancel Icon to delete the relationship we lost the ability to use the onBlur of the text field since clicking the X was dispatching the onBlur after the delete and the state was not being properly set, I Fix that and using the Model state as indicator of focus and that way we can save the relationship name without the use of the button